srcpkgs/crystal/template

```diff
@@ -0,0 +1,127 @@
+# Template file for 'crystal'
+pkgname=crystal
+version=1.0.0
+revision=1
+archs="x86_64* i686*"
+_shardsversion=0.14.1
+_molinilloversion=0.1.0
+_bootstrapversion=1.0.0
+_bootstraprevision=1
+hostmakedepends="which tar llvm11 pkg-config"
+makedepends="gc-devel libatomic_ops pcre-devel libevent-devel libyaml-devel libxml2-devel"
+depends="gc-devel libatomic_ops pcre-devel libevent-devel libyaml-devel libxml2-devel gmp-devel openssl-devel llvm11 gcc pkg-config"
+checkdepends="readline-devel libyaml-devel gmp-devel openssl-devel git"
+short_desc="Crystal Programming Language"
+maintainer="lvmbdv <ata.kuyumcu@protonmail.com>"
+license="Apache-2.0"
+homepage="https://crystal-lang.org/"
+distfiles="
+ https://github.com/crystal-lang/crystal/archive/${version}.tar.gz
+ https://github.com/crystal-lang/shards/archive/v${_shardsversion}.tar.gz
+ https://github.com/crystal-lang/crystal-molinillo/archive/v${_molinilloversion}.tar.gz"
+checksum="6b85c9139a891732c1f10638030635ff9dbe4e6818cad81a5dbd72db40913019
+ 040b080acb4d4d44bfe07897219fdcb7525f783e253d8878d8381bf78f13f328
+ 1ecc7a8bf52a3bfdc0134d4c58f1155ef204a22a3fed151ac2d4ba6a9e9e0a15"
+nocross="FIXME: someone needs to sort out the llvm --cxxflags for cross building"
+_crystalflags="--release --no-debug --progress"
+
+build_options="binary_bootstrap"
+build_options_default="binary_bootstrap"
+desc_option_binary_bootstrap="Bootstrap using precompiled binaries"
+
+if [ "$build_option_binary_bootstrap" ]; then
+	case "$XBPS_MACHINE" in
+	x86_64)
+		distfiles+=" https://github.com/crystal-lang/crystal/releases/download/${_bootstrapversion}/crystal-${_bootstrapversion}-${_bootstraprevision}-linux-x86_64.tar.gz"
+		checksum+=" 00211ca77758e99210ec40b8c5517b086d2ff9909e089400f6d847a95e5689a4"
+		;;
+	x86_64-musl)
+		distfiles+=" https://github.com/crystal-lang/crystal/releases/download/${_bootstrapversion}/crystal-${_bootstrapversion}-${_bootstraprevision}-linux-x86_64.tar.gz"
+		checksum+=" 00211ca77758e99210ec40b8c5517b086d2ff9909e089400f6d847a95e5689a4"
+		;;
```

Comment:
```suggestion
	x86_64*)
		distfiles+=" https://github.com/crystal-lang/crystal/releases/download/${_bootstrapversion}/crystal-${_bootstrapversion}-${_bootstraprevision}-linux-x86_64.tar.gz"
		checksum+=" 00211ca77758e99210ec40b8c5517b086d2ff9909e089400f6d847a95e5689a4"
		;;
```

Reply:
Thanks, for this suggestion. Did not know about these wildcards in a case statement.